src/creatures/players/vip/player_vip.hpp

```diff
@@ -0,0 +1,74 @@
+/**
+ * Canary - A free and open-source MMORPG server emulator
+ * Copyright (©) 2019-2024 OpenTibiaBR <opentibiabr@outlook.com>
+ * Repository: https://github.com/opentibiabr/canary
+ * License: https://github.com/opentibiabr/canary/blob/main/LICENSE
+ * Contributors: https://github.com/opentibiabr/canary/graphs/contributors
+ * Website: https://docs.opentibiabr.com/
+ */
+
+#pragma once
+
+#include "creatures/creatures_definitions.hpp"
+
+class Player;
+
+struct VIPGroup {
+	uint8_t id = 0;
+	std::string name;
+	bool customizable;
```

Comment:
It is also advisable to initialize variable

Reply:
Updated!